applications/luci-app-alist/Makefile

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2023 ImmortalWrt.org
```

Comment:
This might be problematic. I need a second opinion here. @hnyman @jow- @nbd168 @hauke

Reply:
I'm not sure what do you mean by "problematic".
Is it a license issue? This repo is also licensed under Apache-2.0.